wolvic/browser/session_settings.cc

```diff
@@ -71,4 +113,17 @@ std::string SessionSettings::GetDefaultUserAgent(UserAgentMode mode) const {
   }
 }
 
+void SessionSettings::RenderViewHostChanged(content::RenderViewHost* old_host,
+                                            content::RenderViewHost* new_host) {
+  DCHECK_EQ(new_host, web_contents()->GetRenderViewHost());
+
+  //UpdateEverything();
+}
```

Comment:
Empty implementation?

Reply:
This is inspired by the AwSettings class, which updates the UA override whenever the WebContent instance changes. We can refactor our SetUseragenOverride method so that we execute the updating logic from here as well. 

But yeah, for now it's an empty implementation; we can remove it from now if you prefer.